XRTK.Lumin/Packages/com.xrtk.lumin/Profiles/LuminControllerDataProviderProfile.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Controllers;
+using XRTK.Definitions.Controllers.Hands;
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Lumin.Profiles
+{
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Input System/Controller Data Providers/Lumin Controller Data Provider Profile", fileName = "LuminControllerDataProviderProfile", order = (int)CreateProfileMenuItemIndices.Input)]
+    public class LuminControllerDataProviderProfile : BaseMixedRealityControllerDataProviderProfile
+    {
+        #region Global Hand Settings Overrides
+
+        [Header("General Settings")]
+
+        [SerializeField]
+        [Tooltip("If set, hand mesh data will be read and available for visualization. Disable for optimized performance.")]
+        private bool handMeshingEnabled = false;
```

Comment:
We don't need these here now that we split apart the controller and hand providers